shared/ui-components/src/forms/CheckboxGroup.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+import { ErrorMessage } from "./ErrorMessage"
+
+interface CheckboxSingle {
+  id: string
+  label: string
+  defaultChecked?: boolean
+  register: any
+}
+
+interface CheckboxGroupProps {
+  error?: boolean
+  required?: boolean
+  errorMessage?: string
+  name: string
+  groupLabel: string
+  groupNote: string
+  fields: CheckboxSingle[]
+}
+
+export const CheckboxGroup = ({
+  name,
+  groupLabel,
+  fields,
+  required = false,
+  error,
+  groupNote,
+  errorMessage,
+}: CheckboxGroupProps) => {
+  return (
+    <>
+      {groupLabel && (
+        <label className="field-label--caps">{groupLabel}</label>
+      )}
+      {groupNote && (
+        <p className="field-note mb-4">{groupNote}</p>
+      )}
+      <div className="mt-3 field">
```

Comment:
I think if you add the conditional `error` class name (like `<div className={"field " + (error ? "error" : "")}>`) it'll resolve the styling issue @slowbot pointed out